2000-06-21  Jakub Jelinek  <jakub@redhat.com>

	* g++.old-deja/g++.other/stmtexpr2.C: New test.

--- gcc/testsuite/g++.old-deja/g++.other/stmtexpr2.C.jj	Wed Jun 21 10:58:25 2000
+++ gcc/testsuite/g++.old-deja/g++.other/stmtexpr2.C	Wed Jun 21 10:58:50 2000
@@ -0,0 +1,9 @@
+// Build don't link:
+// Special g++ Options: -O2
+// Origin: Jakub Jelinek <jakub@redhat.com>
+
+void bar(int);
+void foo(int x)
+{
+  bar(({ int y; y = x; }));
+}
